documentation/src/main/asciidoc/topics/con_estimating_size_data_set.adoc

```diff
@@ -0,0 +1,44 @@
+[id='data-set-size_{context}']
+= Estimating the size of your data set
+Calculate your total data set to work out the correct heap size per JVM.
+
+As a rough guide, you can estimate the total size of your data set with this formula:
+
+[source,options="nowrap",subs=attributes+]
+----
+Number of entries * (Key size + Value size + Memory overhead)
+----
```

Comment:
@danberindei Should we add in the number of owners here or take that into account?

Reply:
I would say "yes" because otherwise people will wonder why they can only use half of the memory capacity of the cluster?

Reply:
I would say it depends what you calculate.
Total memory for the complete cluster (only if DIST) then yes * numOwners 
If you need the memory per node divide by numNodes after it for DIST caches.
In repl mode each node need that amount of memory.

Reply:
why not just reference https://infinispan.org/docs/stable/titles/tuning/tuning.html#capacity_planning ?